noteable_magics/sql/meta_commands.py

```diff
@@ -676,19 +674,28 @@ def fully_introspect(
         )
 
     def introspect_foreign_keys(
-        self, inspector, schema_name, relation_name
+        self, inspector: SchemaStrippingInspector, schema_name: str, relation_name: str
     ) -> List[ForeignKeysModel]:
         """Introspect all foreign keys for a table, describing the results as a List[ForeignKeysModel]"""
 
         fkeys: List[ForeignKeysModel] = []
 
         fkey_dicts = inspector.get_foreign_keys(relation_name, schema_name)
 
+        # Convert from SQLA foreign key dicts to our ForeignKeysModel. But beware,
```

Comment:
This is the main change of the branch. Others are just code quality cleanups: remove dead code, improve type annotations.